Mono.Cecil.props

```diff
@@ -1,101 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">net_4_0_Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <BuildDirectory Condition=" '$(BuildDirectory)' == '' ">$(MSBuildThisFileDirectory)</BuildDirectory>
-    <FileAlignment>512</FileAlignment>
-    <SchemaVersion>2.0</SchemaVersion>
-    <OutputType>Library</OutputType>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\cecil.snk</AssemblyOriginatorKeyFile>
-    <OutputType>Library</OutputType>
-    <OutputPath>$(BuildDirectory)\bin\$(Configuration)\</OutputPath>
```

Comment:
:memo: The output paths did change in this PR, but that didn't seem to be a particular problem. It's certainly easier to manage the build configurations/frameworks with the default directory layout.

Reply:
It's very likely that is going to impact users that are building from source, which is a common scenario.